<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>-4</y>
      <width>391</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="lessonsTab">
     <attribute name="title">
      <string>Lessons</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Add lesson</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="addLesonInput">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addLessonBtn">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>60</y>
        <width>61</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lessonsList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>371</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="addTab">
     <attribute name="title">
      <string>Add</string>
     </attribute>
     <widget class="QLabel" name="addLessonTitle">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>No lesson </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Word:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="wordInput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="translatedInput">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Translated:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addWordPairBtn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>330</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="trainTab">
     <attribute name="title">
      <string>Train</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="wordsTab">
     <attribute name="title">
      <string>Words</string>
     </attribute>
     <widget class="QListWidget" name="wordsList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>361</width>
        <height>521</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>397</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
